### ISSUE TITLE
Add documentation for new Enable Obsidian Integration settings

### ISSUE THREAD

**Issue by jamesx4** [deathau/markdownload#182]
For the life of me I cannot figure out what is supposed to go in the `Obsidian Folder` setting for the `Enable Obsidian Integration` settings.  And I cannot find any documentation on this feature other than it requires the Advanced URI plugin in Obsidian which I've installed/enabled.  Some documentation in the Markdownload guide would be great!

**Comment by ichmoimeyo** [comment]
In my case on a chromebook Imy default downoad folder is ...

**Folder inside Downloads/ to store MarkDownload clips:**

... underneath I give it my folder path ...

`_alex/!!gDrive local partial/computingz/L_O_LOCAL/O/O_ichmoimeyo/z_capture`

**Comment by aancw** [comment]
Obsidian folder is the folder for where you save the content in Obsidian hierarchy. I already push the patch for this, but still not merged.

**Comment by aancw** [comment]
The Obsidian folder is "Web Clippers"

![Screen Shot 2022-11-16 at 10 56 47](https://user-images.githubusercontent.com/6284204/202080007-437cb0bb-ebbc-4626-8cbf-18e091ffd826.png)

**Comment by jamesx4** [comment]
> In my case on a chromebook Imy default downoad folder is ...
> 
> **Folder inside Downloads/ to store MarkDownload clips:**
> 
> ... underneath I give it my folder path ...
> 
> `_alex/!!gDrive local partial/computingz/L_O_LOCAL/O/O_ichmoimeyo/z_capture`

Yes, I can get it my selected browser text to save in a specific directory under Downloads, but my understanding was that this Obsidian option would allow me to save those clipping directly into my Obsidian vault.  I have yet to get that to work.  

I have a vault named "James" and folder in that vault called "Import".  I configure Markdownloader extension with "James" for the vault and "Import" for the Obsidian Folder setting.  When I do Download Tab as Markdown, it still puts in my Downloads directory.  This is using Microsoft Edge.  But I've also tried Chrome w/o success.

**Comment by ichmoimeyo** [comment]
@jamesx4 

> Yes, I can get it my selected browser text to save in a specific directory under Downloads, but my understanding was that this Obsidian option would allow me to save those clipping directly into my Obsidian vault.

My Obsidian vault IS LOCATED @ 

`_alex/!!gDrive local partial/computingz/L_O_LOCAL/O/O_ichmoimeyo`

... thus my clippings get saved correctly in my vault.

Browsers, chrome etc. generally for security reasons only allow extensions etc. to download stuff into the systems official Download folder.

So to get this to work I believe you'll need to relocate your vault from

Root/James

to

Root/Downloads_Dir/James

**Comment by lazypete365** [comment]
In case anyone finds it helpful, I worked around the vault location issue by creating a symlink to my vault's browser clips inside my Downloads folder (tested to work fine on Firefox and Windows, linux shouldn't have a problem either).

I found that a lot less disruptive than having to relocate the whole vault just for this extension.

**Comment by BryanWall** [comment]
I thought that I had this working previously with Obsidian not located in the Downloads folder, but it's not working now, at least not on Macos + Edge or Chrome. 

Using a Symlink does work on Macos, so I'll use that until this is fixed or better documented.
